app/components/AllOrders.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+export class AllOrders extends React.Component {
+  render () {
```

Comment:
i think we need to pass props as an argument to the `render` function, otherwise you'll hit a "props is undefined" error